### ISSUE TITLE
Can Snipe-IT maintain user clients/hardware/devices?

### ISSUE THREAD

**Issue by passboltUser** [snipe/snipe-it#6340]
Periodically there are maintenance weeks in our company. We select a user client device, check the subitems and click finish. That device is then marked as 'maintained'. Does Snipe-IT offer such a feature?

**Comment by passboltUser** [comment]
@memtech3
> You could do status labels?  Maybe custom fields with regex?

How?

> "In Repair"

I'll have to look where this In Repair is.

I just found `/hardware/maintenances`. So it looks like it has a maintenance feature. I'll check it out.

**Comment by passboltUser** [comment]
Ok, I've checked it out: I went to `/hardware`, clicked an asset, then on top clicked the Maintenance tab, entered maintenance type and date and clicked Save. Then the maintained assets can be seen in `/hardware/maintenances`. But one configure that there's a fixed date until certain assets must be maintained(, otherwise one gets an email notification and these items have then a red color background as to visually note that they are sill not maintained and over schedule)?
